zebra/label_manager.c

```diff
@@ -41,6 +41,23 @@
 
 struct label_manager lbl_mgr;
 
+/* privileges */
```

Comment:
Can we add a bit of a comment here about why we are not using zebra/main.c priviledges?  Might help future people sort it out.  Typically we have 1 priviledge structure per daemon, not 2

Reply:
No reason, actually it is fixed in the next commit. I will rearrange commits in a new PR. Sorry for the confusion.